integrations/fullstory/lib/index.js

```diff
@@ -40,8 +40,11 @@ FullStory.prototype.initialize = function() {
   (function(m,n,e,t,l,o,g,y){
     if (e in m) {if(m.console && m.console.log) { m.console.log('FullStory namespace conflict. Please set window["_fs_namespace"].');} return;}
     g=m[e]=function(a,b,s){g.q?g.q.push([a,b,s]):g._api(a,b,s);};g.q=[];
+    o=n.createElement(t);o.async=1;o.crossOrigin='anonymous';o.src='https://'+_fs_host+'/s/fs.js';
+    y=n.getElementsByTagName(t)[0];y.parentNode.insertBefore(o,y);
```

Comment:
These two lines needn't be added. This integration loads `fs.js` on line 20.

Reply:
removed, and added comment

Reply:
Apologies for the late review @sanjaybv, @butanian, and @mericsson, but some of line 43 here is important, specifically the `crossOrigin` and `async` attributes. Can we move those up to the script tag declaration on line 20?

Reply:
@jameremo The line of code you commented on is from an old commit and the code is out of date. Not sure if this has been addressed or not?